src/components/Home/WorksList/Project/Project.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import data from "../../../../data.json";
+import "./Project.css";
+
+export default function Project(props) {
+    let proj;
+    data.forEach((project) => {
+        if (project.title === props.location.pathname.slice(9)) {
+            return (proj = project);
+        }
+    });
+    let { year, title, img, description } = proj;
+
+    return (
+        <div className="project-card-info-container">
+            <Link to="/">
+                <button>Back to Works</button>
+            </Link>
+            <p>{year}</p>
+            <h1>{title}</h1>
+            <h3>{description}</h3>
+            <div className="project-card-img-container">
+                <picture>
+                    <img
+                        className="project-img-info"
+                        src={"." + img}
```

Comment:
use absolute path .. you also need to change it in the `data.json` .. this line should be `src={img}`

Reply:
if i need to make it only {img} then i need to change it in the data.json to relative path !

Reply:
yes , change it

Reply:
but not relative .. absolute